java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -1559,7 +1559,7 @@ open class KotlinFileExtractor(
                 val setter = p.setter
 
                 if (getter == null) {
-                    if (p.modality != Modality.FINAL || !isExternalDeclaration(p)) {
+                    if (!isExternalDeclaration(p)) {
```

Comment:
Can you add a test case that shows when this change is making a difference? I was the one originally introducing this, and I made the mistake of not adding a test case.

Reply:
I don't have a test case; I'm not aware of a situation in which this code makes any difference.